tests/logs/community_push/test_community_push/test_do_community_push_skip_houly_sap_ha[SAP].txt

```diff
@@ -27,8 +27,8 @@
-[   DEBUG] Enriched push item for aws_storage: AmiPushItem(name='ami_pushitem', state='PENDING', src='/foo/bar/sample_product_test.raw', dest=['fake-destination-access'], md5sum=None, sha256sum=None, origin=None, build=None, build_info=KojiBuildInfo(name='test-build', version='7.0', release='20230101', id=None), signing_key=None, release=AmiRelease(product='SAP', date=datetime.datetime(2023, 12, 12, 0, 0), arch='x86_64', respin=1, version='7.0', base_product='sample_base', base_version='1.0', variant='variant', type='ga'), description='', boot_mode=None, marketplace_title_template=None, marketplace_name=None, type='access', region='fake-destination', virtualization='hvm', volume=None, root_device=None, sriov_net_support=None, ena_support=None, uefi_support=None, billing_codes=AmiBillingCodes(name='Access2', codes=['bp-63a5400a']), release_notes=None, usage_instructions=None, recommended_instance_type=None, marketplace_entity_type='awstest', image_id=None, public_image=False, scanning_port=None, user_name=None, version_title=None, security_groups=[], access_endpoint_url=None)
+[   DEBUG] Mapped push item for aws_storage: AmiPushItem(name='ami_pushitem', state='PENDING', src='/foo/bar/sample_product_test.raw', dest=[Destination(id=None, meta={'billing-code-config': {'sample-hourly': {'name': 'Hourly2', 'codes': ['bp-6fa54006'], 'image_name': 'sample_product', 'image_types': ['hourly']}, 'sample-access': {'name': 'Access2', 'codes': ['bp-63a5400a'], 'image_name': 'sample_product', 'image_types': ['access']}}}, architecture='x86_64', destination='fake-destination-access', overwrite=False, restrict_version=False, restrict_major=None, restrict_minor=None, provider='awstest', tags=None)], md5sum=None, sha256sum=None, origin=None, build=None, build_info=KojiBuildInfo(name='test-build', version='7.0', release='20230101', id=None), signing_key=None, release=AmiRelease(product='SAP', date=datetime.datetime(2023, 12, 12, 0, 0), arch='x86_64', respin=1, version='7.0', base_product='sample_base', base_version='1.0', variant='variant', type='ga'), description='', boot_mode=None, marketplace_title_template=None, marketplace_name=None, type='hourly', region=None, virtualization='hvm', volume='/dev/sda1', root_device=None, sriov_net_support=None, ena_support=None, uefi_support=None, billing_codes=None, release_notes=None, usage_instructions=None, recommended_instance_type=None, marketplace_entity_type=None, image_id=None, public_image=None, scanning_port=None, user_name=None, version_title=None, security_groups=[], access_endpoint_url=None)
```

Comment:
These debug diffs are because of the inclusion of the `volume` property on 765b9d27e615b3911be93133a0b1dca3b2b7b6f3